config/routes.rb

```diff
@@ -1,3 +1,27 @@
 Myflix::Application.routes.draw do
+  root to: "pages#front"
   get 'ui(/:action)', controller: 'ui'
+  get 'home', to: 'videos#index'
+  get 'register', to: 'users#new'
+  get 'sign_in', to: 'sessions#new'
+  post 'sign_in', to: 'sessions#create'
+  get 'sign_out', to: 'sessions#destory'
+
+
+  get 'my_queue', to: 'queue_items#index'
+  post 'my_queue', to: 'queue_items#update_queue'
+
+  resources :videos, only: [:index, :show] do
+    collection do
+      get 'search', to: 'videos#search'
+    end
+    member do
+      resource :reviews, only: [:create]
+    end
```

Comment:
Interesting, you don't need to call member route like this. `resources :reviews, only: [:create]` should be enough to build member for a reviews.